kpops/component_handlers/kafka_connect/kafka_connect_handler.py

```diff
@@ -129,15 +129,15 @@ def __dry_run_connector_deletion(self, connector_name: str) -> None:
                 )
             )
             log.debug(f"DELETE /connectors/{connector_name} HTTP/1.1")
-            log.debug(f"HOST: {self._connect_wrapper.host}")
+            log.debug(f"HOST: {self._connect_wrapper.url}")
         except ConnectorNotFoundException:
             log.warning(
                 f"Connector Destruction: connector {connector_name} does not exist and cannot be deleted. Skipping."
             )
 
     @classmethod
-    def from_pipeline_config(cls, pipeline_config: PipelineConfig) -> Self:
+    def from_pipeline_config(cls, config: KpopsConfig) -> Self:
         return cls(
-            connect_wrapper=ConnectWrapper(host=pipeline_config.kafka_connect_host),
-            timeout=pipeline_config.timeout,
+            connect_wrapper=ConnectWrapper(kafka_connect_config=config.kafka_connect),
```

Comment:
I'd say kwarg is unnecessary
```suggestion
            connect_wrapper=ConnectWrapper(config.kafka_connect),
```